### PR TITLE
Improve section about Exonum Java Binding [ECR-3855].

### PR DESCRIPTION
**Pull request by skletsun**
Improved section about Exonum Java Binding.

### CODE REVIEW

File: src/get-started/java-binding.md [hunk 1]
```diff
@@ -2,8 +2,9 @@
 
 <!-- cspell:ignore testnet,prepend,JDWP -->
 
-**Exonum Java App** is an application that includes the Exonum framework
-and Java services runtime environment.
+**Exonum Java Binding** is a framework for building blockchain applications in Java, powered by Exonum.
```
Comment (skletsun) [inline src/get-started/java-binding.md]: Despite the original part was about the Exonum Java _App_ to me it looks like it should be about Exonum Java _Binding_. Is this correct substitution?

Comment (dmitry-timofeev) [reply src/get-started/java-binding.md]: I reckon yes.

**Comment by dmitry-timofeev** [comment]
@vitvakatu anything to add (that's a copy from our readme)?

**Comment by vitvakatu** [comment]
The next section says about installation of _Exonum Java App_, so I propose to add smth like:
```
Exonum Java Binding provides:
- An SDK for service development as a number of Java libraries. (link to https://exonum.com/doc/api/java-binding/0.9.0-rc2/index.html ?)
- Exonum Java Application - a precompiled Exonum node with built-in support for running Java services.
```

**Pull request merged**